ddtrace/tracer/option.go

```diff
@@ -119,6 +119,12 @@ type config struct {
 	// errors will record a stack trace when this option is set.
 	noDebugStack bool
 
+	// profilerHotspots specifies whether profiler code hotspots is enabled.
```

Comment:
```suggestion
	// profilerHotspots specifies whether profiler code hotspots are enabled.
```
Debatable though 🙂 Or....
```suggestion
	// profilerHotspots specifies whether profiler Code Hotspots is enabled.
```